### PR TITLE
skinny 1.3.19

### PR DESCRIPTION
**Pull request by seratch**
https://github.com/skinny-framework/skinny-framework/releases/tag/1.3.19

### CODE REVIEW

File: Library/Formula/skinny.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Skinny < Formula
   desc "Full-stack web app framework built on Scalatra"
   homepage "http://skinny-framework.org/"
-  url "https://github.com/skinny-framework/skinny-framework/releases/download/1.3.18/skinny-1.3.18.tar.gz"
-  sha256 "03551f3d87d85bbd0ab5ec1279c8ef62db3588921fe6ffca88892a4c896715d1"
+  url "https://github.com/skinny-framework/skinny-framework/releases/download/1.3.19/skinny-1.3.19.tar.gz"
+  sha256 "142c9261c5a1d5d59524b62d229055f7496f3c6a"
```
Comment (DomT4) [inline Library/Formula/skinny.rb]: You've used a `sha1` sum here instead of `sha256`.

Comment (seratch) [inline Library/Formula/skinny.rb]: Fixed.

**Comment by DomT4** [comment]
You're missing a letter from the SHA256 :).

```
Error: SHA256 mismatch
Expected: 1a130086dfacc1719b25871c01a1bf27daeffe27a54940e8d3d90eee79868b29b
Actual: a130086dfacc1719b25871c01a1bf27daeffe27a54940e8d3d90eee79868b29b

==> brew audit skinny							 FAILED
Error: 1 problem in 1 formula
skinny:
 * Stable: sha256 should be 64 characters
```

**Comment by seratch** [comment]
Oops, sorry for bothering you again. I just failed to correctly paste the sha256 :-(

**Comment by DomT4** [comment]
No worries at all, It's an easy mistake we've all fallen into a few times :). Thanks for squashing your commits each time, makes life easier for me when I pull and keeps our commit tree as clean as possible :beers:.

Merged in https://github.com/Homebrew/homebrew/commit/a83c99b7bb743b1ef2dde550c167bb4a5de53634. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**